docs/getting-started/upload-a-directory.md

```diff
@@ -68,3 +68,7 @@ Once your data has been [fully processed into the network](/docs/advanced/tags),
 [https://gateway.ethswarm.org/bzz/b25c89a...214917b/index.html](https://gateway.ethswarm.org/bzz/b25c89a401d9f26811680476619a1eb4a4e189e614bc6161cbfd8b343214917b/index.html)
 
 If you are not able to download your file from a different Bee node, you may be experiencing connection issues, see [troubleshooting connectivity](/docs/installation/connectivity) for assistance.
+
+:::warn
+Not all directory structures, as supported by tar, can be supported in Bee. The differences arises, as not all valid directory structures are valid paths for an HTTP router (e.g. paths which are in a parent directory to the root). Bee will accept and upload the invalid directory structures, but some of your content might not be accesible.
```

Comment:
- broke lines for better diff experience in the future
- which -> that
- accesible -> accessible

```suggestion
Not all directory structures, as supported by tar, can be supported in Bee.
The differences arises, as not all valid directory structures are valid paths
for an HTTP router (e.g. paths that are in a parent directory to the root).
Bee will accept and upload the invalid directory structures, but some of
your content might not be accessible.
```